src/components/definition-list/dl.component.js

```diff
@@ -10,6 +10,10 @@ const Dl = ({
   w = 50,
   dtTextAlign = "right",
   ddTextAlign = "left",
+  pl = "3px",
```

Comment:
Can you add these default values to the tests please https://github.com/Sage/carbon/blob/4e8087fc78051e8b2bd011d9283be3143adf35d4/src/components/definition-list/definition-list.spec.js#L34

Reply:
good point, thanks, added